### ISSUE TITLE
undefined method marked_for_destruction? al crear match

### ISSUE THREAD

**Issue by CrisHD935** [IIC2143/Syllabus#65]
Hola! Me pasa que luego de implementar la asociación entre teams y matches, cuando intento crear un nuevo match me manda el siguiente error
![image](https://github.com/IIC2143/Syllabus/assets/133688044/7523d0d2-6dfc-499d-9bb5-1e5240b31d73)

Busqué en internet y no logré encontrar una solución a este problema :/

**Comment by TheExcel2** [comment]
Creo que tuve el mismo problema, Al crear el match le estás pasando directamente los id de los equipos?

**Comment by CrisHD935** [comment]
Si, eso causa algo?

**Comment by TheExcel2** [comment]
Sí, el tema es que los atributos TeamB y TeamA son references, al crear una instancia de match necesitas pasarle objetos Team, no los id. Aunque ahora que recuerdo bien, el error debiera decirte eso explícitamente, así que no estoy seguro de que sea por eso.

**Comment by 00nrs** [comment]
Tengo el mismo error, ayuda :(

**Comment by AlonsoPizarr** [comment]
Hola, yo también tengo el mismo error

**Comment by jpsandoval** [comment]
Hola, yo vi este error alguna vez. El problema estaba relacionado con del dependent: :destroy .
En el caso que vi, el match tenia un dependent: destroy de los teams o un team tenia un dependent destroy a unos matchs .... (teamA y team B). 

Lo que hice fue borrar el dependent: :destroy, y al eliminar un team buscaba los matchs manualmente y los borraba.

Me avisan si funciona.
